docs/devops/create-dotnet-github-action.md

```diff
@@ -0,0 +1,194 @@
+---
+title: "Tutorial: Create a GitHub Action with .NET"
+description: Learn how to create a GitHub action with a containerized .NET app.
+author: IEvangelist
+ms.author: dapine
+ms.date: 03/19/2021
+ms.topic: tutorial
+---
+
+# Tutorial: Create a GitHub Action with .NET
+
+Learn how to create a .NET app that can be used as a GitHub Action. [GitHub Actions](https://github.com/features/actions) enable workflow automation and composition. With GitHub Actions, you can build, test, and deploy source code from GitHub. Additionally, actions expose the ability to programmatically interact with issues, create pull requests, perform code reviews, and manage branches. For more information on continuous integration with GitHub Actions, see [Building and testing .NET](https://docs.github.com/actions/guides/building-and-testing-net).
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+>
+> - Prepare a .NET app for GitHub Actions
+> - Define action inputs and outputs
+> - Compose a workflow
+
+## Prerequisites
+
+- A [GitHub account](https://github.com/join)
+- The [.NET 5.0 SDK or later](https://dotnet.microsoft.com/download/dotnet)
+- A .NET  integrated development environment (IDE)
+  - Feel free to use the [Visual Studio IDE](https://visualstudio.microsoft.com)
+
+## The intent of the app
+
+The app in this tutorial performs code metric analysis by:
+
+- Scanning and discovering **.csproj* and **.vbproj* project files.
```

Comment:
It works correctly currently, but to guarantee any parser will render it the same:

```suggestion
- Scanning and discovering *\*.csproj* and *\*.vbproj* project files.
```